src/main/resources/META-INF/spring/applicationContext.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8" standalone="no"?>
-<beans xmlns="http://www.springframework.org/schema/beans" xmlns:aop="http://www.springframework.org/schema/aop" xmlns:context="http://www.springframework.org/schema/context" xmlns:jee="http://www.springframework.org/schema/jee" xmlns:tx="http://www.springframework.org/schema/tx" xmlns:util="http://www.springframework.org/schema/util" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.springframework.org/schema/aop               http://www.springframework.org/schema/aop/spring-aop-3.1.xsd               http://www.springframework.org/schema/beans               http://www.springframework.org/schema/beans/spring-beans-3.1.xsd               http://www.springframework.org/schema/context               http://www.springframework.org/schema/context/spring-context-3.1.xsd                       http://www.springframework.org/schema/jee               http://www.springframework.org/schema/jee/spring-jee-3.1.xsd                       http://www.springframework.org/schema/tx               http://www.springframework.org/schema/tx/spring-tx-3.1.xsd               http://www.springframework.org/schema/util               http://www.springframework.org/schema/util/spring-util-3.1.xsd">
+<beans xmlns="http://www.springframework.org/schema/beans" xmlns:aop="http://www.springframework.org/schema/aop" xmlns:context="http://www.springframework.org/schema/context" xmlns:jee="http://www.springframework.org/schema/jee" xmlns:tx="http://www.springframework.org/schema/tx" xmlns:util="http://www.springframework.org/schema/util" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.springframework.org/schema/aop               https://www.springframework.org/schema/aop/spring-aop.xsd               http://www.springframework.org/schema/beans               http://www.springframework.org/schema/beans/spring-beans.xsd               http://www.springframework.org/schema/context               https://www.springframework.org/schema/context/spring-context.xsd                       http://www.springframework.org/schema/jee               https://www.springframework.org/schema/jee/spring-jee.xsd                       http://www.springframework.org/schema/tx               http://www.springframework.org/schema/tx/spring-tx-3.1.xsd               http://www.springframework.org/schema/util               http://www.springframework.org/schema/util/spring-util-3.1.xsd">
```

Comment:
Are we expecting to still be using the same versions of `spring-tx` and `spring-util`? Those stand out to me as still pointing at the same older versions while `spring-aop` and `spring-beans` and `spring-context` have been updated.

Reply:
I'm not actually sure what these do but the build is working and we need to move ahead. Going to leave this as a future cleanup item.